pages/api/chat.ts

```diff
@@ -1,50 +1,21 @@
-import { ChatBody, Message, OpenAIModelID } from "@/types";
-import { DEFAULT_SYSTEM_PROMPT } from "@/utils/app/const";
-import { OpenAIStream } from "@/utils/server";
-import tiktokenModel from "@dqbd/tiktoken/encoders/cl100k_base.json";
-import { init, Tiktoken } from "@dqbd/tiktoken/lite/init";
-// @ts-expect-error
-import wasm from "../../node_modules/@dqbd/tiktoken/lite/tiktoken_bg.wasm?module";
+import { ChatBody } from '@/types';
```

Comment:
It seems that the ChatBody type is the only thing being used from the @/types module. If you are not using anything else from that module, you can remove the import statement to optimize the code.